core/src/main/kotlin/pages/PageNodes.kt

```diff
@@ -119,7 +119,7 @@ class MemberPageNode(
         else MemberPageNode(name, content, dri, documentable, children, embeddedResources)
 }
 
-data class PlatformData(val platformType: Platform, val targets: List<String>) {
+data class PlatformData(val name: String, val platformType: Platform, val targets: List<String>) {
```

Comment:
I wonder whether we should pass the `moduleName` using `PlatformData` or should we have a "global" one in the `context`? @sellophane ?

Reply:
Well, it might be a good idea actually